cmake/FindJpegPkg.cmake

```diff
@@ -0,0 +1,82 @@
+#[==============================================[
+FindJpegPkg
+-----------
+
+Searching jpeg library library and creating imported
+target JPEG::JPEG (this is different from shipped one with CMake)
+
+#]==============================================]
+
+# TODO Append parts for Version compasion and REQUIRED support
+if (MSVC OR MINGW)
+  return()
+endif()
+
+# Try to find JPEG using a module or pkg-config. If that doesn't work, search for the header.
+if (NOT TARGET JPEG::JPEG)
+  find_package(JPEG)
+  if (JPEG_FOUND)
```

Comment:
Modern versions of CMake (starting from 3.12, and which is Travis CI uses) already defines `JPEG::JPEG` imported target here in CMake bundled `FindJPEG.cmake` so we need to check if target `JPEG::JPEG` already defined